$NetBSD: patch-ac,v 1.1.1.1 2008/02/06 04:22:33 bjs Exp $

Move two closed parentheses to their proper places.  The patch-a[de] files
reflect this change.

--- aclocal.m4.orig	2006-08-03 08:16:32.000000000 -0400
+++ aclocal.m4
@@ -195,9 +195,9 @@ AC_DEFUN(AC_CHECK_VA_COPY,[
     dnl #   6. check for assignment approach (assuming va_list is a pointer)
     __va_copy_check(ASP, [do { *(d) = *(s); } while (0)])
     dnl #   7. check for memory copying approach (assuming va_list is a struct)
-    __va_copy_check(CPS, [memcpy((void *)&(d), (void *)&(s)), sizeof((s))])
+    __va_copy_check(CPS, [memcpy((void *)&(d), (void *)&(s), sizeof((s)))])
     dnl #   8. check for memory copying approach (assuming va_list is a pointer)
-    __va_copy_check(CPP, [memcpy((void *)(d), (void *)(s)), sizeof(*(s))])
+    __va_copy_check(CPP, [memcpy((void *)(d), (void *)(s), sizeof(*(s)))])
     if test ".$ac_cv_va_copy" = .; then
         AC_ERROR([no working implementation found])
     fi
